crates/types/src/logs/mod.rs

```diff
@@ -46,6 +46,7 @@ pub use tail::*;
     derive_more::Into,
     Serialize,
     Deserialize,
+    bilrost::Message,
```

Comment:
We also have `BilrostNewType` that we can use instead of `bilrost::Message` that makes sure the inner type is just `flattened` in the parent message. An example of that being used is here 

https://github.com/restatedev/restate/blob/a7884cbe045aaee48877aee5eb7b33dd46a18f82/crates/types/src/identifiers.rs#L52

Reply:
Sorry @lsytj0413 I didn't mention that using BilrostNewType also apply to this one here. We usually use this with rust newtype pattern `struct Newtype(T)` instead of a Message to save couple of bytes.

Las comment I promise :)

Reply:
fixed, ptal @muhamadazmy